bindings/commercetools/commercetools.go

```diff
@@ -0,0 +1,179 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
```

Comment:
After joining CNCF we have shifted to Apache License Version 2.0. Please update.